src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -110,17 +106,17 @@
   }
 
   public List<Either<Command, CodeAction>> computeCodeActions(CodeActionParams params, CancelChecker cancelToken) {
-    List<Either<Command, CodeAction>> codeActions = new ArrayList<>();
-    URI uri = create(params.getTextDocument().getUri());
-    Optional<ProjectBindingWrapper> binding = bindingManager.getBinding(uri);
-    for (Diagnostic d : params.getContext().getDiagnostics()) {
+    var codeActions = new ArrayList<Either<Command, CodeAction>>();
```

Comment:
What is the inferred type of `codeActions` here? I guess it will be `ArrayList` no? I wonder if the previous notation wasn't cleaner, only exposing the interface to downstream code.

Reply:
According to https://openjdk.java.net/projects/amber/LVTIstyle.html, I should no worry too much, and using `var` should be fine :)